packages/components/inputs/radio-input/src/radio-option.js

```diff
@@ -49,19 +52,9 @@ const Option = (props) => {
           <RadioOptionUncheckedIcon size="medium" />
         )}
       </div>
-      <div
```

Comment:
Moved to a styled component: https://github.com/commercetools/ui-kit/pull/1539/files#diff-3b6859bcdf33380029f531b3b281716fR5